src/main/resources/react/src/components/local/en.js

```diff
@@ -16,12 +16,33 @@ const en = {
 		logEmail: 'Please enter a valid username or email',
 		logPassword: 'Invalid password',
 		regEmail: 'Please enter a valid email address',
-		regNick: 'More than 2 and less than 50 characters, no spaces',
+		regNick: 'More than 1 and less than 50 characters, no spaces',
```

Comment:
I didn't catch where such changes listed or discussed? Back-end still used 2 characters as required minimum.
@Jack11M do you know anything about that?

Reply:
Ou, I've got it... "More than 2..." meens At least 3. 
@Wolshebnik just ignore that. 
@Jack11M you misundestood that message as well.